app/views/employments/AddEmploymentNameFormView.scala.html

```diff
@@ -14,35 +14,46 @@
  * limitations under the License.
  *@
 
-@import includes.forms.{input, submit}
-@import includes.{cancelLink, ptaHeader}
-@import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+@import includes.cancelLink
 @import includes.forms.errorSummary
-@import includes.backLink
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
+@this(main: MainTempleteNew, form: FormWithCSRF, govukInput: GovukInput, govukButton: GovukButton, govukErrorSummary: GovukErrorSummary)
 
 @(nameForm: Form[String])(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
-
-@mainTemplate(
-  title = Messages("tai.addEmployment.addNameForm.title"),
-  authedUser = Some(user),
-  articleClasses = None) {
-
-  @backLink()
-  @errorSummary(Messages("tai.income.error.form.summary"), nameForm, Seq("flush--bottom"))
-
-  @form(action = controllers.employments.routes.AddEmploymentController.submitEmploymentName()) {
-    @input(
-      field = nameForm("employmentName"),
-      labelText = Messages("tai.addEmployment.addNameForm.title"),
-      classes = Seq("form-control form-control--full-width"),
-      preHeadingText = Some(Messages("add.missing.employment"))
-    )
-
-    @submit()
-  }
-
-  @cancelLink(url = controllers.employments.routes.AddEmploymentController.cancel())
+@main(
+    title = Messages("tai.addEmployment.addNameForm.title"),
+    authedUser = Some(user)) {
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                @if(nameForm.errors.nonEmpty) {
+                    @govukErrorSummary(ErrorSummary(errorList = nameForm.errors.asTextErrorLinks, title = Text(messages("tai.income.error.form.summary"))))
+                }
+
+                @form(action = controllers.employments.routes.AddEmploymentController.submitEmploymentName()) {
+                    <span class="govuk-caption-xl">
+                        <span class="govuk-visually-hidden">@messages("tai.ptaHeader.accessible.preHeading")</span>
+                        @messages("add.missing.employment")
+                    </span>
+                    @govukInput(Input(
+                        id = "employmentName",
+                        name = "employmentName",
+                        value = nameForm.data.get("employmentName"),
```

Comment:
what happens if this is none?

Reply:
Nothing, the first time is set to empty and if the customer inputs a value and the presses back will see the value written down in the input box